supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/Run.scala

```diff
@@ -132,14 +132,22 @@ case class Run(runId: String,  jobDescriptor: BackendJobDescriptor, genomicsInte
     val currentStatus = status()
 
     if (!(previousStatus contains currentStatus)) {
-      // DONE: PBE: Moved state transition logging and "db" writes to our actor.
-      // Only thing "left" here via another ticket is:
-      // PBE deleted abort function registration
+      val prevStateName = previousStatus map { _.toString } getOrElse "-"
+      // TODO PBE: change println in this file to log statements
+      println(s"Status change from $prevStateName to $currentStatus")
+
+      // PBE deleted db writes for run id and status
+
+      if (currentStatus.isRunningOrComplete && (previousStatus.isEmpty || !previousStatus.get.isRunningOrComplete)) {
+        jobDescriptor.abortFunction = Some(() => abort())
+      }
     }
+
     currentStatus
   }
 
   def abort(): Unit = {
+    println(s"Aborting JES Run: $runId")
```

Comment:
Remove this `println`. Call `logger.info(…)` over in `JesJobExecutionActor.abort`, inside the `foreach` block calling this function.